.github/workflows/test.yml

```diff
@@ -115,24 +153,22 @@ jobs:
           DOCKER_USERNAME: "op://Data Engineering/Dockerhub/username"
```

Comment:
should the 1pw step also have the conditional?

Reply:
Are you asking if it shouldn't? It currently does. 

This whole thing is obviously a little messy - I actually wonder if maybe with the reworking I could just make the jobs conditional instead of all steps in the job. Don't quite remember the specific reason I found that I had to do it that way before, but it's possible I've got around it with where the vars are being set now.